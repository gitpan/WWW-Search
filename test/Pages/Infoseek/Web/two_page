http://www.infoseek.com/Titles?rf=0&qt=Greedo+AND+collectible&col=WW&nh=25&st=0&
http://infoseek.go.com/Titles?rf=0&qt=Greedo+AND+collectible&col=WW&nh=25&st=25&&sf=1
