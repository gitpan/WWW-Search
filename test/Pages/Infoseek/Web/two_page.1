<html><head><base target="_top"><meta name="IS" content="ok_flash"><title>Search results for: Greedo AND collectible</Title></head><body bgcolor="#FFFFFF" link="#0000CC" vlink="#990099">
<table width="100%" cellpadding="0" cellspacing="0" border="0">
<tr><td rowspan="2" width="1%"><a href="/Home/Home.html?svx=search_header&nh=25"><img src="/images/tmpl/search_go.gif" width="120" height="38" border="0" hspace="0" vspace="0"></td>
<td height="23" width="100%">&nbsp;</td></tr>
<tr><td bgcolor="#000000" height="15"><spacer type="block" width="1" height="15"></td></tr>
</table><font face="Helvetica,Arial" size="2"><a href="/Home/Home.html?svx=search_breadcrumb&rf=0&qt=Greedo+AND+collectible&col=WW&nh=25&st=25&&sf=1">GO</a> &gt; <b>Greedo AND collectible</b> <br></font><table width="100%" border="0" cellpadding="0" cellspacing="0" bgcolor="#DEDECA"><tr><td bgcolor="#000000" colspan="3" height="1"><spacer type="block" width="1" height="1"></td></tr><tr><td bgcolor="#999999" colspan="3" height="1"><spacer type="block" width="1" height="1"></td></tr><tr><td bgcolor="#DEDECA" colspan="3">&nbsp;</td></tr><tr><td width="40%" rowspan="3">&nbsp;</td><td nowrap><font face="Helvetica,Arial" size="3"><form action="/Titles" method="get" name="WW_Search"><b>Search</b></font><font face="Verdana,Helvetica,Arial" size="1">&nbsp;<input type="radio" value="WW" name="col" checked><b>New</b> search&nbsp;<input type="radio" value="WW,cat_RES" name="col">Search <b>within</b> results</font><br><img src="/images/clear.gif" width="1" height="1"></td><td width="40%" rowspan="3">&nbsp;</td></tr><tr><td valign="top" nowrap><input type="text" name="qt" value="" size="35" maxlength="511">&nbsp;<INPUT TYPE=hidden NAME="sv" VALUE="IS">
<INPUT TYPE=hidden NAME="lk" VALUE="noframes">
<INPUT TYPE=hidden NAME="nh" VALUE="25">
<input type=hidden name="cc" value="WW">
<input type=hidden name="oq" value="Greedo AND collectible">
<input type="submit" value="Find"></td></tr><tr><td nowrap><font face="Verdana,Helvetica,Arial" size="1"><a href="/find?pg=advanced_www.html&ud9=advanced_www&nh=25">Search&nbsp;options</a>&nbsp;|&nbsp;<a href="/Help?pg=SearchTips.html&nh=25">How to search</a>&nbsp;|&nbsp;</font><br><img src="/images/clear.gif" height="4" width="1"></td></tr></form><!-- End Search Box --><tr><td colspan="3" height="8"><spacer type="block" width="1" height="8"></td></tr></table><table border="0" cellpadding="0" cellspacing="0" width="100%"><tr><td bgcolor="#000000" height="1"><spacer type="block" width="1" height="1"></td></tr></table>
<img src="/images/clear.gif" border="0" width="1" height="6"><br>
<center><table width="100%" cellpadding="0" cellspacing="0" border="0"><tr><td align="center" valign="top"><table align="left" width="121" height="60" cellpadding="1" cellspacing="0" border="0">
<tr><td bgcolor="#000000" valign="top">
<table width="119" height="58" cellpadding="0" cellspacing="0" border="0">
<tr><td bgcolor="#FFFFFF" align="center" valign="bottom" width="119" NOWRAP><font face="Arial,Helvetica" size="-2">Books at</font></td></tr>
<tr><td bgcolor="#FFFFFF" align="center" valign="bottom" width="119" NOWRAP><font face="Verdana,Arial,Helvetica" size="-1"><b><a href="/redirect?sv=IS&lk=noframes&ak=BORD&rl=LINK-C0-standard-flash-borders&_nr=1&rd=http://www.borders.com/cgi-bin/partner?PARTNER_ID=101010097&Url=http://www.borders.com/infoseek/section_10000.html">Borders.com</a></b></font></td></tr>
<tr><td bgcolor="#FFFFFF" align="center" valign="bottom" width="119" NOWRAP><font face="Arial,Helvetica" size="-2">In stock, Fast delivery.</font></td></tr>
<tr><td bgcolor="#FFFFFF" align="center" valign="bottom" width="119" NOWRAP><a href="/redirect?sv=IS&lk=noframes&ak=BORD&rl=LINK-C0-standard-flash-borders&_nr=1&rd=http://www.borders.com/cgi-bin/partner?PARTNER_ID=101010097&Url=http://www.borders.com/infoseek/section_10000.html"><img src="/images/partners/borders/031999borders_logo.gif" width="119" height="16" border="0"></a><br></td></tr>
</table>
</td></tr>
</table>
</td><td>&nbsp;&nbsp;</td><td valign="top" align="center" colspan="3"><a href="http://www.infoseek.com/redirect?sv=IS&lk=&tid=&ak=4410-15872-1016-GEN&rd=http://www.prizepoint.com/welcome.asp?RC=720" target="_top"><img border=1 width=468 height=60 src="/ads/44101016.gif" alt="[Click here to win an MP3 Player!]"><br><FONT FACE="verdana,arial,helvetica" SIZE="1"><b>Click here</b></FONT></a></td></tr></table></center><br>
<table width="100%" border="0" cellpadding="0" cellspacing="0" bgcolor="#003366">
<tr><td valign="middle" align="left" nowrap colspan="3"><font face="Helvetica,Arial" size="3" color="#FFFFFF"><a name="search">&nbsp;<b>Web search results</b>&nbsp;&nbsp;&nbsp;&nbsp;<font face="Helvetica,Arial" size="2">26 - 35 of <b>35</b> results most relevant to <b>Greedo AND collectible</b> </font>&nbsp;</a></font></td>
<td height="18" valign="middle" align="right" nowrap  colspan="1">&nbsp;
</td></tr>
<tr><td valign="middle" align="left" nowrap colspan="4" bgcolor="#000000"><img src="/images/clear.gif" border="0" width="100%" height="1"></td>
</table>
<spacer type="block" width="1" height="2"><!-- Begin RHS -->
<table cellpadding="1" cellspacing="0" border="0" width="150" align="right" bgcolor="#FFFFFF"><tr><td height="4"><spacer type="block" width="1" height="4"></td></tr><tr><td></td></tr><tr><td><table cellpadding="1" cellspacing="0" border="0" width="150" align="right" bgcolor="#000000"><tr><td><table cellpadding="6" cellspacing="0" border="0" width="148" align="right" bgcolor="#FFFFFF"><tr><td><center><font size="2" face="Verdana,Helvetica,Arial"><b>Buy the book</b></font></center><br><!-- Infoseek cut-->
<FONT FACE="Arial,Helvetica" SIZE="2">Win a guitar signed <br>by <A HREF="/redirect?sv=IS&lk=&ak=BORD&rl=LINK_C99999_UNMAPPED_RESULTS_GO&rd=http://go.borders.com/fcgi-bin/part?PID=122010891&PKIND=aa&PAGE=http%3A%2F%2Fgo%2Eborders%2Ecom%2Fpromo%2Fmclachlan%2Findex%2Excv">Sarah Mclachlan</A>!
<BR><A HREF="/redirect?sv=IS&lk=&ak=BORD&rl=LINK_C99999_UNMAPPED_RESULTS_GO&rd=http://go.borders.com/fcgi-bin/part?PID=122010892&PKIND=aa&PAGE=http%3A%2F%2Fsearch%2Eborders%2Ecom%2Fboutiques%2Fstarwars%2Findex%2Excv">Star Wars</A> must-haves!
<br>Visit our <a href="/redirect?sv=IS&lk=&ak=BORD&rl=LINK_C99999_UNMAPPED_RESULTS_GO&rd=http://go.borders.com/fcgi-bin/part?PID=122010894&PKIND=aa&PAGE=http%3A%2F%2Fsearch%2Eborders%2Ecom%2Fgc%2Findex%2Excv">Gift Center</a>!
<BR>Celebrate <a href="/redirect?sv=IS&lk=&ak=BORD&rl=LINK_C99999_UNMAPPED_RESULTS_GO&rd=http://go.borders.com/fcgi-bin/part?PID=122010893&PKIND=aa&PAGE=http%3A%2F%2Fsearch%2Eborders%2Ecom%2Ffcgi%2Dbin%2Fdb2www%2Fbgilistitems%2Ed2w%2Freport%3FListRef%3D2206">Gay Pride Month</A>!
<br>Oprah's <a href="/redirect?sv=IS&lk=&ak=BORD&rl=LINK_C99999_UNMAPPED_RESULTS_GO&rd=http://go.borders.com/fcgi-bin/part?PID=122010990&PKIND=aa&PAGE=http%3A%2F%2Fsearch%2Eborders%2Ecom%2Ffcgi%2Dbin%2Fdb2www%2Flist%2Ed2w%2Freport%3FListRef%3D1250%26deptCode%3Dbo">Book Club&reg;</a>!
</font>
<!-- Infoseek cut--></td></tr></table></td></tr></table></td></tr></table>
<!-- End RHS -->
<font face="Helvetica,Arial" size="2"><b><a href="http://infoseek.go.com/Titles?rf=0&qt=Greedo+AND+collectible&col=WW&nh=25&st=0&&sf=1"> < Previous 25</a></b>  <a href="http://infoseek.go.com/Titles?rf=0&qt=Greedo+AND+collectible&col=WW&nh=20&st=25&&sf=1&ud4=1">Hide summaries</a>&nbsp;&nbsp;|&nbsp;&nbsp; <a href="http://infoseek.go.com/Titles?rf=0&qt=Greedo+AND+collectible&col=WW&nh=25&st=25&&sf=3">Sort by date</a>&nbsp;&nbsp;|&nbsp;&nbsp;<a href="http://infoseek.go.com/Titles?rf=0&qt=Greedo+AND+collectible&col=WW&nh=25&st=25&&sf=0">Group results</a></font><br><br><font face="Helvetica,Arial" size="2"><b><a href="http://members.aol.com/SweetJezus/swfigs2.htm">The Comic Vault Star Wars Action Figures Page</a></b><br>Action Figures  Welcome!  This is newly renovated to combine the various sections of my action figure pages into one.  WillyAmes@aol.com is doing some incredible work. Check below for a ...<br><font face="Helvetica,Arial" size="2">25% &nbsp;<b>Date:  4 Feb 1998</b>, &nbsp;Size 12.9K, &nbsp;http://members.aol.com/SweetJezus/swfigs2.htm&nbsp;</font><br><a href="http://infoseek.go.com/Titles?rf=0&col=WW&nh=25&&sf=1&cat=RES&fsd=73464968925537203&fs=http%3A//members.aol.com/SweetJezus/swfigs2.htm&svx=find_similar">Find similar pages</a><p><b><a href="http://members.tripod.com/~kpikemusic/starwars/starwarscd.html">Star Wars Trilogy CD</a></b><br>Star Wars Trilogy CD<br><font face="Helvetica,Arial" size="2">25% &nbsp;<b>Date: 23 Jul 1998</b>, &nbsp;Size 1.8K, &nbsp;http://members.tripod.com/~kpikemusic/starwars/starwarscd.html&nbsp;</font><br><a href="http://infoseek.go.com/Titles?rf=0&col=WW&nh=25&&sf=1&cat=RES&fsd=73464968925041056&fs=http%3A//members.tripod.com/%7Ekpikemusic/starwars/starwarscd.html&svx=find_similar">Find similar pages</a><p><b><a href="http://www.wfp51744.w1.com/actfig.htm">Science Fiction Collectibles - Action Figures</a></b><br>Science Fiction Collectibles stocks action figures of characters from Star Trek, Star Wars, and Babylon 5.<br><font face="Helvetica,Arial" size="2">24% &nbsp;<b>Date: 14 May 1999</b>, &nbsp;Size 18.8K, &nbsp;http://www.wfp51744.w1.com/actfig.htm&nbsp;</font><br><a href="http://infoseek.go.com/Titles?rf=0&col=WW&nh=25&&sf=1&cat=RES&fsd=260927303411934161&fs=http%3A//www.wfp51744.w1.com/actfig.htm&svx=find_similar">Find similar pages</a><p><b><a href="http://www.spacestar.net/users/rbig/jediknights/actionfigures/figurelist.htm">Star Wars Collectibles</a></b><br>Powers of the Force 2  Deluxe Series  Shadows of the Empire  Power F/X  Beast/Deluxe Beast Series  Exclusives  Placo Key Chains  *Designates Scheduled Release  Powers of the Force 2.  QTY.  FIGURE  WAVE/  COLLECTION  RELEASE  DATE  2-1B Medic Droid ...<br><font face="Helvetica,Arial" size="2">22% &nbsp;<b>Date: 14 Feb 1998</b>, &nbsp;Size 31.3K, &nbsp;http://www.spacestar.net/users/rbig/jediknights/actionfigures/figurelist.htm&nbsp;</font><br><a href="http://infoseek.go.com/Titles?rf=0&col=WW&nh=25&&sf=1&cat=RES&fsd=289637751035561910&fs=http%3A//www.spacestar.net/users/rbig/jediknights/actionfigures/figurelist.htm&svx=find_similar">Find similar pages</a><p><b><a href="http://lightningc.com/oldstarwarstoys.html">Orginal Kenner Star Wars Toys Listings</a></b><br>Lightning <b style="background:#FFFF99">Collectible's</b> selection of Orginal Kenner Star Wars Figures.  Over 200 figures in stock.  All credit card accepted! Order on-line with our easy shopping cart.  We want to be your source ...<br><font face="Helvetica,Arial" size="2">21% &nbsp;<b>Date: 16 Jan 1999</b>, &nbsp;Size 27.2K, &nbsp;http://lightningc.com/oldstarwarstoys.html&nbsp;</font><br><a href="http://infoseek.go.com/Titles?rf=0&col=WW&nh=25&&sf=1&cat=RES&fsd=289637751059696164&fs=http%3A//lightningc.com/oldstarwarstoys.html&svx=find_similar">Find similar pages</a><p><b><a href="http://www.users.mis.net/~lmk9/">index6</a></b><br>top This site is owned by . Want to join the ? You are Smurf Visitor Number Since May 18, 1997 !!! Please sign my !! Page Created: May 17, 1997 Last Revised: October 9, 1997; 2:13 p.m. "Hi" and thanks for stopping by my Smurf ...<br><font face="Helvetica,Arial" size="2">21% &nbsp;<b>Date: 19 Nov 1997</b>, &nbsp;Size 22.0K, &nbsp;http://www.users.mis.net/~lmk9/&nbsp;</font><br><a href="http://infoseek.go.com/Titles?rf=0&col=WW&nh=25&&sf=1&cat=RES&fsd=289637751043610102&fs=http%3A//www.users.mis.net/%7Elmk9/&svx=find_similar">Find similar pages</a><p><b><a href="http://members.tripod.com/~Greenea/index-3.html">Action Figures(loose)</a></b><br>A variety of carded action figures<br><font face="Helvetica,Arial" size="2">19% &nbsp;<b>Date: 18 Feb 1999</b>, &nbsp;Size 28.9K, &nbsp;http://members.tripod.com/~Greenea/index-3.html&nbsp;</font><br><a href="http://infoseek.go.com/Titles?rf=0&col=WW&nh=25&&sf=1&cat=RES&fsd=289637751060632283&fs=http%3A//members.tripod.com/%7EGreenea/index-3.html&svx=find_similar">Find similar pages</a><p><b><a href="http://www.thecantina.simplenet.com/news.htm">Kenner News</a></b><br><br><font face="Helvetica,Arial" size="2">19% &nbsp;<b>Date: 24 Jan 1999</b>, &nbsp;Size 40.0K, &nbsp;http://www.thecantina.simplenet.com/news.htm&nbsp;</font><br><a href="http://infoseek.go.com/Titles?rf=0&col=WW&nh=25&&sf=1&cat=RES&fsd=289637751059975731&fs=http%3A//www.thecantina.simplenet.com/news.htm&svx=find_similar">Find similar pages</a><p><b><a href="http://www.diamondcomics.com/sp_catalogs/figure_factory/figurefactory_cat.txt">Figure Factory ALIENS & UFO'S Safari Ltd. ALIEN SHIPS Based upon documented encounters, ...</a></b><br><br><font face="Helvetica,Arial" size="2">19% &nbsp;<b>Date:  2 Nov 1998</b>, &nbsp;Size 78.8K, &nbsp;http://www.diamondcomics.com/sp_catalogs/figure_factory/figurefactory_cat.txt&nbsp;</font><br><a href="http://infoseek.go.com/Titles?rf=0&col=WW&nh=25&&sf=1&cat=RES&fsd=73464968945415769&fs=http%3A//www.diamondcomics.com/sp_catalogs/figure_factory/figurefactory_cat.txt&svx=find_similar">Find similar pages</a><p><b><a href="http://members.aol.com/Medolls/co1.html">Medolls' Consignment Dolls,Etc</a></b><br><br><font face="Helvetica,Arial" size="2">17% &nbsp;<b>Date: 26 Dec 1998</b>, &nbsp;Size 25.8K, &nbsp;http://members.aol.com/Medolls/co1.html&nbsp;</font><br><a href="http://infoseek.go.com/Titles?rf=0&col=WW&nh=25&&sf=1&cat=RES&fsd=289637751059705618&fs=http%3A//members.aol.com/Medolls/co1.html&svx=find_similar">Find similar pages</a></font><br clear=all><br clear=all><font face="Helvetica,Arial" size="2"><b><a href="http://infoseek.go.com/Titles?rf=0&qt=Greedo+AND+collectible&col=WW&nh=25&st=0&&sf=1"> < Previous 25</a></b>  </font><br><br><center><p><center><a href="http://www.infoseek.com/redirect?sv=IS&lk=&tid=&ak=4410-15872-1016-GEN&rd=http://www.prizepoint.com/welcome.asp?RC=720" target="_top"><img border=1 width=468 height=60 src="/ads/44101016.gif" alt="[Click here to win an MP3 Player!]"><br><FONT FACE="verdana,arial,helvetica" SIZE="1"><b>Click here</b></FONT></a></center><p><center>
<table cellpadding="0" cellspacing="0" border="0" width="100%"><tr><td bgcolor="#DEDECA">&nbsp;</td></tr></table><p><table border="0" cellpadding="0" cellspacing="0" width="100%"><tr><td rowspan="4" width="35%" align="center" valign="top"><font size="2" face="Verdana,Helvetica,Arial"><a href="/Titles?nh=25&tv=brag&qt=Greedo+AND+collectible&col=WW&sv=IS&svx=brag_page"><img src="/images/misc/powered_by_stack.gif" border="0" alt="Powered by Infoseek"><br>What makes us better</a></font></td><tr><td width="40%" rowspan="3">&nbsp;</td><td nowrap><font face="Helvetica,Arial" size="3"><form action="/Titles" method="get" name="WW_Search"><b>Search</b></font><font face="Verdana,Helvetica,Arial" size="1">&nbsp;<input type="radio" value="WW" name="col" checked><b>New</b> search&nbsp;<input type="radio" value="WW,cat_RES" name="col">Search <b>within</b> results</font><br><img src="/images/clear.gif" width="1" height="1"></td><td width="40%" rowspan="3">&nbsp;</td></tr><tr><td valign="top" nowrap><input type="text" name="qt" value="" size="35" maxlength="511">&nbsp;<INPUT TYPE=hidden NAME="sv" VALUE="IS">
<INPUT TYPE=hidden NAME="lk" VALUE="noframes">
<INPUT TYPE=hidden NAME="nh" VALUE="25">
<input type=hidden name="cc" value="WW">
<input type=hidden name="oq" value="Greedo AND collectible">
<input type="submit" value="Find"></td></tr><tr><td nowrap><font face="Verdana,Helvetica,Arial" size="1"><a href="/find?pg=advanced_www.html&ud9=advanced_www&nh=25">Search&nbsp;options</a>&nbsp;|&nbsp;<a href="/Help?pg=SearchTips.html&nh=25">How to search</a>&nbsp;|&nbsp;</font><br><img src="/images/clear.gif" height="4" width="1"></td></tr></form><!-- End Search Box --></table><br><br><center><font size="2" face="Verdana,Helvetica,Arial">If there is a site you would like listed, click <a href="/AddUrl?pg=SubmitUrl.html&nh=25&svx=search_footer"><b>add URL</b></a>.<hr><br><center><b>GO Network partners:</b>&nbsp;&nbsp;<a href="http://abc.go.com/"><b>ABC.com</b></a>&nbsp;&nbsp;<a href="http://abcnews.go.com"><b>ABCNEWS.com</b></a>&nbsp;&nbsp;<a href="http://disney.go.com"><b>Disney.com</b></a>&nbsp;&nbsp;<a href="http://espn.go.com"><b>ESPN.com</b></a>&nbsp;&nbsp;<a href="http://family.go.com"><b>Family.com</b></a>&nbsp;&nbsp;<a href="http://mrshowbiz.go.com"><b>Mr. Showbiz</b></a><br><br><a href="/doc?pg=copyright.html">Copyright</a> &copy; 1999 Infoseek Corporation.  All rights reserved.&nbsp;&nbsp;&nbsp;&nbsp;<a href="mailto:comments@infoseek.com">Contact us</a>&nbsp;&nbsp;&nbsp;&nbsp;<a href="http://info.go.com/doc/policy/disclaimer.html">Legal Notices</a>&nbsp;&nbsp;&nbsp;&nbsp;<a href="http://info.go.com/doc/policy/privacy.html">Privacy policy</a></font><br><br><br></center></body></html><pre>









</pre>
</body>
</html>
