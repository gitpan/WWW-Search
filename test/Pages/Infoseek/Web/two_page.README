This set of test-result pages was created on Wed Jun 30 11:43:09 1999
