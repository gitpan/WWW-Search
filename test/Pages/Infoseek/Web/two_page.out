http://www.1mainstreet.net/greedo.html
http://www.ct-info.com/co/empire/starwars.html
http://www.entertainmentearth.com/
http://www.tias.com/stores/fitinc/starwars-1.html
http://members.tripod.com/~jeff19/collection.html
http://www.toysrgus.com/images-nosta/index.html
http://ourworld.compuserve.com/homepages/stephane_faucourt/SWcollec.htm
http://www.1mainstreet.net/starwars.html
http://www.nightmarefactory.com/starwars.html
http://classic-collectibles.com/12_StarWars%20Figures.htm
http://www.westfield.com/wow/starwars.html
http://www.info2000.net/~jlee/huntquar.htm
http://www.drtonguestoys.com/swcatalog.htm
http://user.mc.net/ncshimp/starwars.htm
http://www.rcboyd.com/starwars/index.htm
http://bradystoys.com/Kenner/StarWars.htm
http://members.aol.com/tusktrader/index.htm
http://www.classic-collectibles.com/star_wars_action_figures_green_normal_picture.htm
http://www.classic-collectibles.com/star_wars_action_figures_orange_normal_picture.htm
http://www.thecomicsource.com/text/ccgsw.html
http://ourworld.compuserve.com/homepages/stephane_faucourt/swpotf95.htm
http://www.tncuniverse.com/tnclist.htm
http://members.aol.com/SweetJezus/swfigs2.htm
http://members.tripod.com/~kpikemusic/starwars/starwarscd.html
http://www.wfp51744.w1.com/actfig.htm
http://www.spacestar.net/users/rbig/jediknights/actionfigures/figurelist.htm
http://lightningc.com/oldstarwarstoys.html
http://www.users.mis.net/~lmk9/
http://members.tripod.com/~Greenea/index-3.html
http://www.thecantina.simplenet.com/news.htm
http://www.diamondcomics.com/sp_catalogs/figure_factory/figurefactory_cat.txt
http://members.aol.com/Medolls/co1.html
